Description: Use system jQuery instead of bundled copy
 Replace the bundled jquery-3.7.1.min.js with a reference to the
 system-installed jQuery from libjs-jquery package.
Author: Archisman Panigrahi <apandada1@gmail.com>
Forwarded: not-needed
Last-Update: 2025-12-21

--- a/typhoon/typhoon.html
+++ b/typhoon/typhoon.html
@@ -136,7 +136,7 @@
     <button class="btn">TRY AGAIN</button>
 </div>
 </div>
-<script src="jquery-3.7.1.min.js"></script>
+<script src="file:///usr/share/javascript/jquery/jquery.min.js"></script>
 <script src="script.js"></script>
 </body>
 </html>
--- a/meson.build
+++ b/meson.build
@@ -12,7 +12,7 @@
 
 # Install the entire media directory (recursively)
-install_subdir('typhoon', install_dir: data_dir)
+install_subdir('typhoon', install_dir: data_dir, exclude_files: ['jquery-3.7.1.min.js'])
 
 # Create a wrapper script in /usr/bin

